xline/src/server/xline_server.rs

```diff
@@ -71,6 +73,12 @@ pub struct XlineServer {
     compact_cfg: CompactConfig,
     /// Server timeout
     server_timeout: ServerTimeout,
+    /// Client tls config
+    #[cfg(not(madsim))]
+    client_tls_config: Option<ClientTlsConfig>,
```

Comment:
Same as above, should it be a client config?

Reply:
Same as above